packaging/ubuntu-14.04/snapd.postrm

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+# note that we run the debhelper scripts first to ensure snapd is stopped
+#DEBHELPER#
```

Comment:
IIRC lxd tries to do fancy stuff with snapd API in the stop-command. Maybe this is what is causing issues here.